vocab/properties.json

```diff
@@ -1720,7 +1655,7 @@
         ]
     },
     "studiedState": {
-        "description": null,
+        "description": "Reference to a point in time and condition in which something or someone was studied.",
```

Comment:
That's not true. The time is actually the essential keyword here. Condition could be even deleted because that is what is depending on the time. What about:

"Reference to a point in time at which something or someone was studied in a particular mode or condition." ??

Reply:
Sounds good :)